### ISSUE TITLE
Is there any wrappers for maxdet problem?

### ISSUE THREAD

**Issue by TongCHEN779** [cvxopt/CVXOPT.jl#8]
Hello, is it possible to use your wrapper to solve problems whose objectives are log_det or det_rootn, etc? Thanks in advance.

**Comment by martinandersen** [comment]
No, unfortunately not. That would require that the interface is extended to include a wrapper for [cvxopt.solvers.cp](http://cvxopt.org/userguide/solvers.html#cvxopt.solvers.cp).
